site/app/views/submission/HomeworkView.php

```diff
@@ -67,6 +68,32 @@ public function showGradeable($gradeable, $late_days_use, $extensions) {
 </div>
 HTML;
         }
+        $ldu = new LateDaysCalculation($this->core, $gradeable->getUser()->getId());
+        $late_days_data = $ldu->getGradeable($gradeable->getUser()->getId(), $gradeable->getId());
+        $late_days_remaining = $late_days_data['remaining_days'];
```

Comment:
Late days remaining is not necessarily the number of late days allowed for an assignment. This is misleading.